server/deployment/src/main/java/io/quarkiverse/openapi/server/generator/deployment/codegen/ApicurioOpenApiServerCodegen.java

```diff
@@ -94,7 +95,12 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
 
     private File convertToJSON(Path yamlPath) throws CodeGenException {
         try {
-            ObjectMapper yamlReader = new ObjectMapper(new YAMLFactory());
+            LoaderOptions loaderOptions = new LoaderOptions();
+            loaderOptions.setCodePointLimit(12 * 1024 * 1024); // 12 MB
+            YAMLFactory yamlFactory = YAMLFactory.builder()
+                    .loaderOptions(loaderOptions)
+                    .build();
+            ObjectMapper yamlReader = new ObjectMapper(yamlFactory);
```

Comment:
You are changing the factory on the server side. This was not in the scope of the original issue. It's not a problem, so I invoke @carlesarnal to take a look and assess the impact. Won't the problem on the client side? 🤔